src/optimization/inlineConstructors.ml

```diff
@@ -184,7 +184,7 @@ let inline_constructors ctx original_e =
 		PMap.find s io.io_fields
 	in
 	let alloc_io_field_full (io:inline_object) (fname:string) (constexpr_option:texpr option) (t:t) (p:pos) : inline_var =
-		let v = alloc_var VInlined fname t p in
+		let v = alloc_var (VInlinedConstructorVariable "") fname t p in
```

Comment:
This looks like it should either stay `VInlined` or become `VInlinedConstructorVariable fname`.

Reply:
I'm hesitating about it, because I overwrite it at the end `v.v_kind <- if (was_user iv) then VInlinedConstructorVariable vname else VInlined;`
I don't know what's the impact of it on `get_pretty_name`, will test!

Reply:
Keep `VInlined` here seems simpler for later tag rewrite (no need to detect when we should put `VInlined` - without overwrite `VGenerated`). Done.